product_docs/docs/pgd/4/harp/07_harp_proxy.mdx

```diff
@@ -144,3 +144,65 @@ This software is designed to be launched through systemd or in a container
 as a top-level process. This also means output is directed to STDOUT and STDERR
 for capture and access through journald or an attached container terminal.
 
+## Leadership consensus storage
+
+Routing decisions are stored in a DCS (Distributed Consensus Store). Harp Manager supports two DCS stores, the BDR Postgres database and etcd. The etcd option is capable of managing a very high volume of the simple transactions used to track leadership leases and lease renewals. It is also lightweight, leaving more server capacity to the core database.
```

Comment:
This is already talked about in the DCS section. Also "The etcd option is capable of managing a very high volume of the simple transactions used to track leadership leases and lease renewals. It is also lightweight, leaving more server capacity to the core database." is complete bull. The etcd is not lightweight and given we normally install it along with the database it actually eats resources from it.

Reply:
At minimum, I'd remove the marketing crap around etcd.

Reply:
Removed.